chapters/functions.tex

```diff
@@ -1228,8 +1228,10 @@ \section{Derivatives and Inverses of Functions}\label{derivatives-and-inverses-o
 
 \begin{nonnormative}
 This means that the function is at least C\textsuperscript{smoothOrder}.
-{\lstinline!smoothOrder = 1!} means that the function can be differentiated at least once in order that all output arguments are still continuous, provided the input arguments are continuous.
-If a tool needs the derivative of a function, e.g.\ for index reduction or to compute an analytic Jacobian, the function can be differentiated analytically at least {\lstinline!smoothOrder!} times.
+
+When a tool computes the derivative of a function, e.g., for index reduction or to compute an analytic Jacobian, each differentiation of a function reduces the \lstinline!smoothOrder! by 1.
+The \lstinline!smoothOrder! information can then be used to infer continuity of the resulting differentiated function calls.
+This is a conservative check, however, meaning that a tool may be able to establish continuity of a function call even though the \lstinline!smoothOrder! has been reduced to less than 0.
```

Comment:
```suggestion
The \lstinline!smoothOrder! information can then be used to infer continuity of the resulting differentiated function calls, provided the input arguments are continuous.
This is a conservative check, however, meaning that a tool may be able to establish continuity of a function call even though the \lstinline!smoothOrder! has been reduced to less than 0, and/or some input arguments are not continuous.
```
The previous statement about continuity of input arguments is important and shall not be removed.

Reply:
Sure, I think it disappeared because I thought it goes without saying.  No big deal to put it back.